recipes/appligator/recipe.yaml

```diff
@@ -0,0 +1,42 @@
+context:
+  version: 0.0.1
+  python_min: 3.10
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.

```suggestion
```